### PR TITLE
Log executor exceptions

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: resolwe/flow/executors/__main__.py [hunk 1]
```diff
@@ -56,17 +83,24 @@ async def run_executor():
 
 
 if __name__ == "__main__":
+    # initialize logging ASAP.
```
Comment (dblenkus) [inline resolwe/flow/executors/__main__.py:86]: Capitalize "Initialize".

File: resolwe/flow/executors/__main__.py [hunk 2]
```diff
@@ -28,14 +28,41 @@
...
 from .logger import configure_logging
-from .protocol import ExecutorFiles
-from .transfer import transfer_data
+from .protocol import ExecutorProtocol
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    """Log unhandled exceptions."""
+    message = "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+    logger.error("Unhandled exception in executor: {}".format(message),)
```
Comment (dblenkus) [inline resolwe/flow/executors/__main__.py:45]: Would it make sense to put a new line after colon?

Comment (gregorjerse) [reply resolwe/flow/executors/__main__.py:45]: Yes, and I have removed colon too.

File: resolwe/flow/executors/__main__.py (hunk 1 above)
Comment (gregorjerse) [reply resolwe/flow/executors/__main__.py:86]: OK.

**Pull request merged**